AGENTS.md

```
@@ -98,15 +141,122 @@ if not thread or thread.user_id != current_user.id:
     )
 ```
 
-## Testing Patterns
+ ## Testing Patterns
```

markdownlint-cli2 (0.20.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)